index.html

```diff
@@ -2049,9 +2049,10 @@ <h3 id="Schema-Building">Schema Building:</h3>
     </p>
 
     <p id="Schema-foreign">
-      <b class="header">foreign</b><code>table.foreign(column))</code>
+      <b class="header">foreign</b><code>table.foreign(~mixed~)</code>
```

Comment:
Could we change this to plural columns instead of `~mixed~` ? I know it doesn't make much sense seeing as it accepts both Array and Non-Array string literals, but I ask this to keep the consistency of the documentation as much as possible.

IE, `.dropForeign` right below states `columns`